pr-deploy/src/github.ts

```diff
@@ -24,19 +24,17 @@ const ACTION: Octokit.ChecksUpdateParamsActions = {
   identifier: 'deploy-me-action',
 };
 
-const CHECK_NAME: string = 'pr-deploy-check';
+const check_name = process.env.GH_CHECK;
```

Comment:
Update the readme/example files with these new env variables

Reply:
K added an `.env.example` file, let me update the README entirely in another PR